jupyterhub/tests/test_services_auth.py

```diff
@@ -217,7 +219,8 @@ def finish_thread():
         assert auth.login_url in r.headers['Location']
 
 
-def test_service_cookie_auth(app, mockservice_url):
+def test_hubauth_cookie(app, mockservice_url):
```

Comment:
Min, Do we want to start moving to pytest's parameterize to test specific names? It might make for easier additions to the tests going forward and simplify testing when trying to find a bug as we could more easily drop in individual test cases.